include/linker/kobject-text.ld

```diff
@@ -1,8 +1,8 @@
 #ifndef KOBJECT_TEXT_AREA
 #if defined(CONFIG_DEBUG) || defined(CONFIG_STACK_CANARIES)
-#define KOBJECT_TEXT_AREA	256
+#define KOBJECT_TEXT_AREA	512
 #else
-#define KOBJECT_TEXT_AREA	128
```

Comment:
If this issue only affects the -O0 case, why are we increasing the size for the optimized case as well?